terra_notebook_utils/drs.py

```diff
@@ -121,6 +121,16 @@ def access(drs_url: str,
                                 key=info.key,
                                 sa_credentials=info.credentials,
                                 requester_pays_user_project=billing_project)
+        # one last sanity check
+        response = requests.get(url, headers={'Range': 'bytes=0-1'})
+
+    if response.status_code >= 400:
+        raise DRSResolutionError(f'Could not generate a valid signed URL for {drs_url} !  Make sure you have access '
```

Comment:
@DailyDreaming - nice job with this error message, thanks!
I like it except for the exclamation point. Please remove that. :)

Reply:
Ha, alright, thanks for the review.